pkg/collector/process_test.go

```diff
@@ -492,11 +363,11 @@ func TestTLSCollectingProcess(t *testing.T) {
 	collectorAddr := cp.GetAddress()
 	go func() {
 		roots := x509.NewCertPool()
-		ok := roots.AppendCertsFromPEM([]byte(fakeCACert))
+		ok := roots.AppendCertsFromPEM([]byte(test.FakeCACert))
```

Comment:
We can also replace TLS related certs like this file in `pkg/exporter/process_test.go` and `pkg/test/exporter_collector_test.go`

Reply:
done